champions.html.slim

```diff
@@ -0,0 +1,77 @@
+---
+layout: microsite
+hero_class: champions
+---
+
+= define 'hero'
+  .row
+    .large-12.column
+      img(src=cdn(site.base_url + "/images/champions/roadtoawesome_graphic.png"))
+    .large-12.column
+      img(src=cdn(site.base_url + "/images/champions/jbosschampion_logo.png"))
+      p JBoss Champions is a selected group of community members who are passionate advocate of JBoss technologies under a program sponsored by Red Hat. JBoss Champions use a variety of JBoss technologies and actively share their deep technical expertise about it in the community, within their company and their customers or partners. This could be done in forums, blogs, screencasts, tweets, conferences, social media, whitepapers, articles, books, and other means.
+
+.wide.microsite-text-points
+  .row
+    .large-12.column
+      h2 Who is a JBoss Champion?
+      p JBoss Champions areas of contributions include, but not limited to:
+      ul
+        li Senior developers, architects, consultants, academia who are using and promoting JBoss technologies
```

Comment:
@adelasofia please can you take a look at this? 

Look at: https://www.dropbox.com/s/q37hc4qzn42ub3k/Screenshot%202015-02-09%2011.36.20.png?dl=0. Notice that the word "promoting" is not horizontally aligned with the word "Senior".

I tried this, but couldn't get it to work: http://stackoverflow.com/questions/11556493/2nd-line-in-li-start-under-the-bullet-after-css-reset